hashes/kubernetes

```diff
@@ -0,0 +1,3 @@
+a53dc25aa091075d73b7f8ae9a518c019a16b821bce6cfe3ff191cf4098df8851de03ec0a9292e534045323b8300532a23e6caceae3e6dfaea00471eac16a223  k8s-1.21.6.tar.gz
+aafb8a0f3613e7511854ed972004a6c5db72fede5f1289ba14044e44c36957ff53332b0ae36762137fd32218db0435449bbb1f49a1e8d50deba8a9a7aa6f480f  kubeadm.amd64
```

Comment:
I just realized the kubeadm binaries don't have versions in the name. I'll go ahead and fix this.